core/src/main/scala/scalaz/MonadTrans.scala

```diff
@@ -15,6 +15,9 @@ trait MonadTrans[F[_[_], _]] {
   /** The [[scalaz.Monad]] implied by this transformer. */
   implicit def apply[G[_] : Monad]: Monad[F[G, ?]]
 
+  def mapF[G[_], A, B](fa: F[G, A])(f: A => G[B])(implicit M: Monad[G]): F[G, B] =
+    Monad[F[G, ?]].bind(fa)(a => liftM(f(a)))
+
```

Comment:
FYI

`liftM` is a hangover from Haskell, in the days before `Applicative` was known about. The `M` there denotes that it is the `Monad` version of `fmap`, contrasted to the one on `Functor`.

You'll also find `liftA` in Haskell, starting at the time `Applicative` was found to be useful.